lib/ts/nextjs.ts

```diff
@@ -127,6 +128,141 @@ export default class NextJS {
             });
         };
     }
+
```

Comment:
also we can introduce a createNewSession helper as explained here: https://github.com/supertokens/docs/pull/765#discussion_r1433744179

Reply:
Added `withPreParsedRequestResponse` function instead.